debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/AlterTableParserListener.java

```diff
@@ -122,6 +125,7 @@ public void exitColumnDefinition(MySqlParser.ColumnDefinitionContext ctx) {
                 else {
                     // all columns parsed
                     // reset global variables for next parsed statement
+                    columnEditors.forEach(columnEditor -> tableEditor.addColumn(columnEditor.create()));
```

Comment:
What's the purpose of this, given it's set to `null` on the next line?

Reply:
@gunnarmorling The editors created by `ALTER TABLE` are added to the table.

Reply:
Hum, I'm still confused. What bug is fixed exactly by this? +1 for extracting it and making it more clear. I've pushed one more commit for the change discussed on the chat; can you rework the PR to split up that first commit here? In general let's keep unrelated things in separate commits for the sake of comprehensibility :)